test/performance/benchmarks_test.go

```diff
@@ -0,0 +1,152 @@
+// +build performance
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package performance
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	pkgTest "github.com/knative/pkg/test"
+	"github.com/knative/pkg/test/ingress"
+	"github.com/knative/serving/test"
+	"github.com/knative/test-infra/shared/junit"
+	"github.com/knative/test-infra/shared/loadgenerator"
+	"github.com/knative/test-infra/shared/testgrid"
+)
+
+const (
+	reqTimeout    = 30 * time.Second
+	app = "helloworld"
+)
+
+var loads = [...]int32{1, 100}
```

Comment:
```suggestion
var loads = []int32{1, 100}
```
arrays are cumbersome in Go :(

Reply:
Could you elaborate the benefits?  I basically want a constant rather than a variable when it is defined globally like this.  [...]type{values} is the closest to define an constant list that I know.

Reply:
It's still not a constant, but it's cumbersome to pass them around. not a big problem here, but we don't use arrays in general.

Reply:
Acknowledged.  I'd like to keep it close to an immutable list.  It is global and does not get passed around.